assets/user-profile/actions.ts

```diff
@@ -258,20 +258,19 @@ export function saveFolder(folder: ITopicFolder, data: {name: string}, global?:
         const state = getState();
         const url = getFoldersUrl(state.company, state.user?._id, global, folder._id);
 
+        const returnFolder = (allFolders: IFoldersUnified, id: string) =>
```

Comment:
It's not a good practice to use the word `return` in the function name, because it causes confusion with the built-in return operator. Use get/find instead. I'd call it simply `getFolder`